en/clinic-user-guide.md

```diff
@@ -0,0 +1,674 @@
+  tidbclusters.pingcap.com                       []                 []              [get list]
+  tidbmonitors.pingcap.com                       []                 []              [get list]
+```
+
+> **Note:**
+>
+> If the cluster meets the criteria of least privilege deployment, you can use a smaller set of privileges. For more information, see [Least privilege deployment](#step-3-deploy-a-diag-pod).
+
+Follow these steps to check the user access:
+
+1. Check the user's *Role* and *clusterRole*:
+
+    ```shell
+    kubectl describe rolebinding -n ${namespace} | grep ${user_name} -A 7
+    kubectl describe clusterrolebinding -n ${namespace} | grep ${user_name} -A 7
+    ```
+
+2. Check the user's access of *Role* and *Cluster Role*:
+
+    ```shell
+    kubectl describe role ${role_name} -n ${namespace}
+    kubectl describe clusterrole ${clusterrole_name} -n ${namespace}
+    ```
+
+### Step 2: Log in to the Clinic Server and get an access token
+
+When Diag uploads data, the access token is used to identify the user and ensures that the data from Diag is uploaded to the organization created by the user. You need to log in to the [Clinic Server](https://clinic.pingcap.com.cn/portal/#/login) to get a token.
+
+1. Sign up and log in to the Clinic Server.
+
+    Go to [Clinic Server](https://clinic.pingcap.com.cn/portal/#/login) and select **Sign in with AskTUG** to enter the AskTUG community login page. If you do not have an AskTUG account, you can create one on that page.
+
+2. Create an organization.
+
+    Create an organization on the Clinic Server. Organization is a collection of TiDB clusters. You can upload diagnostic data on the created organization.
+
+3. Get an access token.
+
+    To get a token, enter the organization page and click the icon in the lower-right corner of the Clusters page, and select **Get Access Token For Diag Tool**. Make sure that you have copied and saved the displayed token information.
+
+    ![An example of a token](/media/clinic-get-token.png)
+
+    > **Note:**
+    >
+    > For security reasons, Clinic Server only displays the token upon the token creation. If you have lost the token, delete the old token and create a new one.
+
+### Step 3: Deploy a Diag Pod
+
+Depending on the network connection of the cluster, you can choose one of the following methods to deploy a Diag Pod:
+
+- Quick online deployment: If the cluster has Internet access and you would like to use the default Diag configuration, it is recommended to use the quick online deployment.
+- Regular online deployment: If the cluster has Internet access and you need to customize the Diag configuration, it is recommended to use the regular online deployment.
+- Offline deployment: If the cluster cannot access the Internet, you can use the offline deployment.
+- Least privilege deployment: If all nodes in the cluster are running under the same namespace, you can deploy Diag to the namespace of the cluster to have the least privileges.
```

Comment:
```suggestion
- Least privilege deployment: If all nodes in the cluster are running under the same namespace, you can deploy Diag to the namespace of the cluster so that Diag has the least privileges.
```